web/src/app/models/submission/submission.model.ts

```diff
@@ -21,28 +21,28 @@ import {Job} from 'app/models/job.model';
 
 import {Result} from './result.model';
 
-export class Submission {
+export class SubmissionData {
```

Comment:
Hu @amysorto thanks for the fix! I think a `Submission` is still a submission, but it has data associated with it.  If we rename it, submissiondata has data, which doesn't sound right. Can we revert this change?